EIPS/eip-6789.md

```diff
@@ -0,0 +1,113 @@
+---
+eip: 6789
+title: Rename gas to mana
+description: This EIP suggests renaming gas to mana, as proposed by Vitalik Buterin in 2015
+author: Pascal Caversaccio (@pcaversaccio)
```

Comment:
IMO Vitalik should have been listed as co-author as the idea originated from him.

Reply:
It's a bit of a gray area. I would like him to be listed as a co-author, but I also feel that he should explicitly consent to being a co-author.

CC @vbuterin in case he responds.